app/src/components/ScheduleBuilder.tsx

```diff
@@ -206,12 +203,12 @@ export default class ScheduleBuilder extends Component<Props, States> {
                         }
                         disabled={false}
                     >
-                        {this.state.filters?.terms.map((t) => {
+                        {this.state.filters?.terms.map((t, idx) => {
                             const { term, year } = SOC_Generic.decodeTermString(
                                 t.CODE,
                             );
                             return (
-                                <option value={t.CODE}>
+                                <option value={t.CODE} key={idx}>
```

Comment:
This is fine because `terms[]` never changes.

I would, however, prefer to use `t.CODE` for each because it is also temporally unique (unique across time) and identifies what the option refers to.